site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -581,10 +581,20 @@ components:
                   a certificate but then extract all relevant data and store
                   it in the properties described above.
                   Either this property or `public-key` must be set.
+            "auto-provisioning-enabled":
```

Comment:
IMHO the management API should return an error if a client tries to register a trust anchor with this property set to `true` but the registry implementation doesn't support auto-provisioning.